### ISSUE TITLE
 Keras symbolic input/output error

### ISSUE THREAD

**Issue by BazilaAfridi** [dhyeythumar/PPO-algo-with-custom-Unity-environment#3]
![image](https://user-images.githubusercontent.com/69035494/151803982-2615903e-0208-4626-9404-3c10ed5d1c04.png)

Can anyone please help me to solve this error?

**Comment by dhyeythumar** [comment]
Hi @BazilaAfridi,

I think this issue is related to installing wrong version of TensorFlow or Keras. I have committed requirements.txt file which will help you to install the exact versions I have used.

**Also note that I have used Python 3.8 to create the environment.**

**Comment by BazilaAfridi** [comment]
Ok sir thank you so so much ☺️. Sir by running train.py file it only
trained single agent for about ten minutes. Wanted to ask that this repo is
still related to one agent and doesn't support multi agents? If yes then
according to your YouTube video how you trained multi agents parallel and
they were performing so well?

Thanks again sir,
Regards,
Bazila

**Comment by dhyeythumar** [comment]
Hi Brazila,
This repo was an attempt to implement PPO algorithm on Unity based environments.

**The code will run for what you have defined in max steps:**
https://github.com/dhyeythumar/PPO-algo-with-custom-Unity-environment/blob/e43f9cd7e3c6c878b836c9819a8187b7820bb69f/train.py#L30-L35

The above configs will only run for 50 steps.

I did had plans to update it for multi agents envs but for now that development is halted.
But to train Multi agent environment & training simultaneous copies of the environment I have used [ML-Agents](https://github.com/Unity-Technologies/ml-agents) repo.
